### DISCUSSION TITLE
RTL layout support

### DISCUSSION THREAD

**Discussion by danielmayost** [AvaloniaUI/Avalonia#7670] [General]
Is there a plan to add RTL layout support to version 11? (#7345)

**Comment by maxkatz6** [comment]
@Gillibald RTL support is already part of master branch (so it will be in 11 version), right? Or is there any more work planned for that? Aside from bugfixes.

**Reply by robloo** [reply]
My limited understanding was FlowDirection is only supported on TextBlock and TextBox in master. That means it isn't full support as some other controls need to invert coordinates and adjust layout for RTL.

**Reply by danielmayost** [reply]
rtl **layout** support don't mplemented at all
see: https://github.com/AvaloniaUI/Avalonia/issues/7345
